doc/examples/viz_ui.py

```diff
@@ -4,11 +4,10 @@
 User Interfaces
 ===============
 
-This example shows how to use the UI API.
-Currently includes button, textbox, panel, and line slider.
+This example shows how to use the UI API. We will demonstrate how to create
+several DIPY UI elements, then use a list box to toggle which element is shown.
 
 First, a bunch of imports.
-
 """
 import os
```

Comment:
can you separate `"""` and `import os` by an empty line. 

We need it to render the documentation correctly